src/asciidoctor-core-api.js

```diff
@@ -359,10 +359,7 @@ AbstractNode.$$proto.getAttribute = function (name, defaultValue, inherit) {
  */
 AbstractNode.$$proto.isAttribute = function (name, expectedValue, inherit) {
   var value = this['$attr?'](name, expectedValue, inherit);
-  if (value === Opal.nil) {
-    return undefined;
-  }
-  return value;
+  return value === Opal.nil ? false : value;
```

Comment:
Is this behavior consistent with the Asciidoctor core API ? We return a boolean or a value (which could be an instance instance of another type) ?

Reply:
Technically, core only returns truthy or falsy values, but I think the public-facing API here can be more precise. We should coerce a truthy value to true too. I'll update.

Reply:
Actually, all we need is a test. It's already returning true and false in all instances as it stands.

Reply:
Perfect then 👌